src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -4496,6 +4497,15 @@ AlterExpression AlterExpression():
        ["=" { alterExp.setUseEqual(true);} ]
        sk3 = RelObjectName() {alterExp.addParameters(sk3); }
     )
+    |
+    (<K_RENAME> {alterExp.setOperation(AlterOperation.RENAME);}
+      (
```

Comment:
@wumpz not entirely sure if my syntax here is the best or not, if you have a resource you recommend for learning javacc let me know. I referenced this part [https://javacc.github.io/javacc/tutorials/lookahead.html#choice-points], so the choice point is on `RENAME`.

Reply:
Unfortunately, I have no such resource. I used to documentation, examples and other JavaCC community comments.